### PR TITLE
Task 9 LikeSVG

### LINKED ISSUE #9: Add the like svg
Create a component from the attached svg.
[like.svg.zip](https://github.com/vbetivu-ellation/crunchygram/files/7286266/like.svg.zip)

### PR DESCRIPTION
**Pull request by MihaelaCatan04**
Added Like SVG Task 9, with height and width properties into the SVG tag.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import styles from "./App.module.css";
 import { Text } from "./components/common";
 import StoreProvider from "./store";
 import Counter from "./components/Counter";
+import LikeSvg from "./components/like_svg";
```
Comment (darsenii-crunchyroll) [inline src/App.js]: Please remove LikeSvg from App.js. We do it only to test our components. We should leave it untouched at the moment in order for other girls to not have conflicts

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (darsenii-crunchyroll) [inline .vscode/launch.json]: Please do not commit this file

File: src/App.js [hunk 3]
```diff
@@ -22,6 +23,7 @@ function App() {
           >
             Learn React
           </a>
+          <LikeSvg />
```
Comment (darsenii-crunchyroll) [inline src/App.js]: Here as well, remove as long as we don't need it

File: src/components/common/like_svg.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const LikeSvg = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" width = "50px" height = "50px" viewBox="0 0 24 24" role="img">
```
Comment (darsenii-crunchyroll) [inline src/components/common/like_svg.js]: Please use 24px for the width and height

File: src/components/common/like_svg.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    <path  d="M19.84 9.476C20.442 6.858 19.07 5 16.965 5c-1.31 0-2.377.534-3.337 1.71L12.046 8.65l-1.542-1.97C9.602 5.53 8.536 5 7.144 5 5.132 5 3.658 7.07 4.21 9.477c.601 2.623 3.21 5.702 7.901 9.099 4.512-3.103 7.054-6.163 7.73-9.1zM16.965 3c3.279 0 5.741 2.935 4.824 6.924-.866 3.77-4.103 7.463-9.71 11.076-5.761-3.962-9.034-7.654-9.819-11.076C1.426 6.285 3.818 3 7.144 3c1.322 0 2.485.352 3.49 1.055l-.105.127.282.002c.456.346.879.766 1.267 1.262a7.499 7.499 0 0 1 1.264-1.236l.31.003a9.964 9.964 0 0 0-.115-.146C14.549 3.356 15.692 3 16.965 3z" />
+  </svg>
+);
+
+export default LikeSvg;
```
Comment (darsenii-crunchyroll) [inline src/components/common/like_svg.js]: Please add an empty line in the end of the component

File: src/components/common/like_svg.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
```
Comment (darsenii-crunchyroll) [inline src/components/common/like_svg.js]: Please rename the file to LikeSvg.js

**Review by ileahu-ellation — APPROVED** [review]
LGTM

**Pull request merged**